extensions/amp-story/1.0/test/test-amp-story-interactive.js

```diff
@@ -284,5 +289,36 @@ describes.realWin(
 
       expect(percentages4).to.deep.equal([33, 33, 33]);
     });
+
+    it('should update store service when selecting option', async () => {
+      const trigger = env.sandbox.stub(
+        ampStoryInteractive,
+        'updateStoryStoreState_'
```

Comment:
Let's not stub/mock private properties, it reduces the scope of the test and makes internal refactoring harder.
Instead, can you get the STATE from the store and assert that it looks right?